include/deal.II/base/parameter_handler.h

```diff
@@ -1423,9 +1431,15 @@ class ParameterHandler : public Subscriptor
    * Print parameters to a logstream. This function allows to print all
    * parameters into a log-file. Sections will be indented in the usual log-
    * file style.
+   *
+   * Before printing, all current parameters and subsections are sorted
+   * alphabetically by default.
+   * This option can be disabled setting the last parameter @p sort_alphabetical
+   * to <kbd>false</kbd>: in this case entries are printed in the same order
+   * as they have been declared.
    */
   void
-  log_parameters(LogStream &out);
+  log_parameters(LogStream &out, const bool &sort_alphabetical = true);
```

Comment:
Do you need this functionality in log_parameters?

Reply:
The only reason to have it there is backward compatibility!